variation-commons-core/src/main/java/uk/ac/ebi/eva/commons/core/models/VariantClassifier.java

```diff
@@ -33,20 +35,28 @@
      * @return Type of the variant
      */
     public static VariantType getVariantClassification(String reference, String alternate, int subSNPClass) {
-        String alphaRegEx = "[A-Z]+";
+        Pattern alphaRegExPattern = Pattern.compile("[A-Z]+");
 
         reference = reference.trim();
         alternate = alternate.trim();
 
-        if (!reference.equals("NOVARIATION")) {
+        boolean isRefAlpha = alphaRegExPattern.matcher(reference).matches();
```

Comment:
Move those two statements closer to the place where they are being used (line 57 or 58) in order to:
- Keep the scope of the variables as reduced as possible
- Avoid having out of context lines to make the code more readable
- Avoid checking the regular expression when is not necessary